### DISCUSSION TITLE
Is it possible to change the default font(s)?

### DISCUSSION THREAD

**Discussion by vincentcomfy** [prasanthrangan/hyprdots#1804] [Q&A] [answered]
Hello,

I managed to  modify a lot of settings by trawling through the existing discussions but I don't see any mention of how fonts are handled. I specifically want to change my default font to Open Sans. Can someone please point me in the right direction?

**Comment by kRHYME7** [comment, marked as answer]
In your userprefs.conf add an exec = command 

The command will be the gsettings command. 

You can check the correct syntax in `~/.config/hypr/themes/common.conf`

**Reply by vincentcomfy** [reply]
I really appreciate you giving me the answer in a way that I could organically figure it out, thank you so much.
Just to confirm, do I need to add all of the lines that are in common.conf?

I assume it's just the font-name, document-font-name and monospace-font-name?

**Reply by vincentcomfy** [reply]
Disregard, I just changed the parameters to test it out and it looks like it worked. It hasn't applied for all apps but I'm guessing they have their own theming. I'll leave it for now.
